Formula/pyside.rb

```diff
@@ -40,26 +40,20 @@ def install
   end
 
   test do
-    system Formula["python@3.9"].opt_bin/"python3", "-c", "import PySide2"
-    system Formula["python@3.9"].opt_bin/"python3", "-c", "import shiboken2"
+    system Formula["python@3.9"].opt_bin/"python3", "-c", "import PySide6"
+    system Formula["python@3.9"].opt_bin/"python3", "-c", "import shiboken6"
 
     modules = %w[
       Core
       Gui
-      Location
-      Multimedia
       Network
       Quick
       Svg
       Widgets
       Xml
     ]
 
-    # QT web engine is currently not supported on Apple
-    # silicon. Re-enable it once it has been enabled in the qt.rb.
-    modules << "WebEngineWidgets" unless Hardware::CPU.arm?
```

Comment:
Is there no `WebEngineWidgets` for pyside6?

Reply:
see the blog [post](https://www.qt.io/blog/add-on-support-in-qt-6.0-and-beyond), the` WebEngineWidgets` still in progress, and will release as additional libraries.